src/app/components/TextField/index.js

```diff
@@ -0,0 +1,28 @@
+import React, { forwardRef } from 'react';
+import PropTypes from 'prop-types';
+
+import styles from './styles.module.scss';
+
+const TextField = forwardRef(({ title, inputVariant, error, ...rest }, ref) => (
```

Comment:
By convention use a function instead of an arrow function 
```
 function nameComponent({props}, ref) {}
 
 export default forwardRef(nameComponent)
```

Reply:
Warning: forwardRef render functions do not support propTypes or defaultProps. Did you accidentally pass a React component?

thats why I use arrow function

Reply:
Oh, you're right

Reply:
Or you could write it as usual and declare the in the props interface
`inputRef?: string & ((instance: HTMLInputElement | null) => void);`
and send `inputRef={register...`